### PR TITLE
Elrond ESDT tokens support

### PR DESCRIPTION
**Pull request by AlexandruPislariu**

### CODE REVIEW

File: packages/cryptoassets/data/esdt.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module.exports = [
+  ['USDC', '555344432d633736663166', 18, '3045022100f03b1ec0d83d8d75d436f688c2007ffb0c71b39a4f72aba036062ea8d7ade99402203449426b83203617279c04ccb00387a77deffc468c27015bfc1590f1950edde9', "WrappedUSDC"],
+  ['MEX', '4d45582d343535633537', 18, '3044022076b7216cc07b827d5dec0468db31d9393b0028dcd5c81bcdc4f5371e0386061f022013f51d51c1421fbf67771a1edc1f607d750cf366392c5c09216fecb61e350f73', 'MEX'],
+  ['RIDE', '524944452d376431386539', 18, '3045022100afa25bb834c8a2a6e7f6e15f3f318e9da3b7991f2a3f9c2f58d73c700be08736022066d3d0ef00e754e050a4b1ba8455129e83dd01c2a8a6b79738373a463d10fe82', 'holoride'],
+]
```
Comment (gre) [inline packages/cryptoassets/data/esdt.js:5]: are these tokens understood by the device? in clear sign i mean

**Review by gre — CHANGES_REQUESTED** [review]
FAIL packages/hw-app-elrond/tests/Elrond.test.ts
  ● provideESDTInfo



      33 |   this.expectedAPDU = expected;
      34 |   this.gotAPDU = got;
    > 35 |   this.stack = new Error().stack;
         |                ^
      36 | }
      37 |
      38 | RecordStoreWrongAPDU.prototype = new Error();

      at new RecordStoreWrongAPDU (packages/hw-transport-mocker/src/RecordStore.ts:35:16)
      at RecordStore.Object.<anonymous>.RecordStore.replayExchange (packages/hw-transport-mocker/src/RecordStore.ts:127:17)
      at TransportReplayer.Object.<anonymous>.TransportReplayer.exchange (packages/hw-transport-mocker/src/openTransportReplayer.ts:43:39)
      at TransportReplayer.<anonymous> (packages/hw-transport/src/Transport.ts:204:33)
      at step (packages/hw-transport/lib/Transport.js:33:23)
      at Object.next (packages/hw-transport/lib/Transport.js:14:53)
      at packages/hw-transport/lib/Transport.js:8:71
      at Object.<anonymous>.__awaiter (packages/hw-transport/lib/Transport.js:4:12)
      at TransportReplayer.Transport.send (packages/hw-transport/lib/Transport.js:110:20)

File: packages/cryptoassets/data/esdt.js (hunk 1 above)
Comment (AlexandruPislariu) [reply packages/cryptoassets/data/esdt.js:5]: Yes, all of these tokens are supported by ledger device already.

**Comment by AlexandruPislariu** [comment]
I removed the hw-app-elrond package implementation since it is already implemented in ledger-live-common repository.
We decided to move forward with that [version](https://github.com/LedgerHQ/ledger-live-common/tree/master/src/families/elrond/hw-app-elrond).

**Review by gre — COMMENTED** [review]
-

**Review by gre — APPROVED** [review]

**Pull request merged**